### PR TITLE
Add MarkerMAG package

### PR DESCRIPTION
**Pull request by songweizhi**
MarkerMAG: linking MAGs with 16S rRNA marker genes using paired-end short reads

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by songweizhi** [comment]
@BiocondaBot please add label

File: recipes/markermag/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/MarkerMAG-{{ version }}.tar.gz
+  sha256: a5a87c5dd319130fb47f7105759e1e48c3906f3e0ce8533241cb2a1d75d97882
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
```
Comment (linzhi2013) [inline recipes/markermag/meta.yaml:16]: have a look at https://docs.conda.io/projects/conda-build/en/latest/user-guide/wheel-files.html and https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html. Usually, we use  " script: python -m pip install --no-deps --ignore-installed ."

"We strongly recommend using the --no-deps option in the pip install command.

If you run pip install without the --no-deps option, pip will often install dependencies in your conda recipe and those dependencies will become part of your package. This wastes space in the package and increases the risk of file overlap, file clobbering, and broken packages.

**Comment by songweizhi** [comment]
@BiocondaBot please add label

**Review by linzhi2013 — APPROVED** [review]

**Comment by linzhi2013** [comment]
@BiocondaBot please merge

**Pull request merged**